Dockerfile

```diff
@@ -14,4 +14,28 @@ ENTRYPOINT poetry run gunicorn --workers=2 "todo_app.app:create_app()" --bind 0.
 
 FROM base as development
 RUN poetry install
-ENTRYPOINT poetry run flask run --host=0.0.0.0
\ No newline at end of file
+ENTRYPOINT poetry run flask run --host=0.0.0.0
+
+FROM base as test
+RUN poetry install
+RUN poetry add selenium
```

Comment:
This line is unnecessary - you're already tracking selenium in your pyproject.toml

Reply:
I'm thinking of keeping it since I'm scared of breaking something

Reply:
A bad practice to get into! Now that you have this pipeline you can safely change it - if it causes issues then your pipeline will fail & tell you, and worst-comes-to-worst you can just add it back 😄